lib/spec/simpleofflinecache.sw-spec.js

```diff
@@ -101,4 +104,36 @@ describe('SimpleOfflineCache instance', function () {
     });
   });
 
+  if (!isChrome) {
+    it('should respect ignoreSearch when matching', function(done) {
+      var cacheName = 'test';
+      var options = {
+        ignoreSearch: true,
+        ignoreMethod: false,
+        ignoreVary: false,
+        prefixMatch: false,
+        cacheName: cacheName
```

Comment:
I read the spec, but leave it just in case the behavior will change. But you are right, will remove it if right now is doing nothing.